matterbridge-bot/README.md

```diff
@@ -45,7 +45,7 @@ helmfile status
 | fullnameOverride | string | `""` | The full release name override |
 | image.pullPolicy | string | `"IfNotPresent"` | The pullPolicy used when pulling the image |
 | image.repository | string | `"42wim/matterbridge"` | The repository of the image |
-| image.tag | string | `"latest"` | The tag of the image. Overrides the image tag whose default is the chart appVersion. |
+| image.tag | string | `"stable"` | The tag of the image. Overrides the image tag whose default is the chart appVersion. |
```

Comment:
nit: If you're setting default to `stable` in values.yaml , then chart version will never default to chart appVersion

Reply:
Unfortunately, their repo only have a few dynamic tags